src/pages/ProductDetail/Count.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import './Count.scss'
+
+const Count = ({ id, quantity, setQuantity }) => {
+  const handleCount = value => {
+    if (quantity + value < 1) return
+    setQuantity(quantity + value)
+  }
+  return (
+    <div className="count" id={id}>
```

Comment:
```suggestion
    <div className="count" >
```
id에 데이터를 전달해주지 않아도 될 것 같네요!